webui/react/src/components/UserSelectFilter.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { text } from '@storybook/addon-knobs';
+import React, { useEffect } from 'react';
+
+import Users from 'contexts/Users';
+import { AuthDecorator, UsersDecorator } from 'storybook/ContextDecorators';
+
+import UserSelectFilter from './UserSelectFilter';
+
+export default {
+  component: UserSelectFilter,
+  decorators: [ AuthDecorator, UsersDecorator ],
+  title: 'UserSelectFilter',
+};
+
+interface Props {
+  value?: string;
+}
+
+const UserSelectFilterWithUsers: React.FC<Props> = ({ value }: Props) => {
+  const setUsers = Users.useActionContext();
+
+  useEffect(() => {
+    setUsers({
+      type: Users.ActionType.Set,
+      value: {
+        data: [
+          { id: 1, isActive: true, isAdmin: true, username: 'admin' },
+          { id: 2, isActive: true, isAdmin: false, username: 'user' },
+          { id: 3, isActive: false, isAdmin: false, username: 'inactive' },
+        ],
+        errorCount: 0,
+        hasLoaded: true,
+        isLoading: false,
+      },
+    });
+  }, [ setUsers ]);
+
+  return (
+    <UserSelectFilter
+      value={value}
```

Comment:
**suggestion:** Thanks for making `onChange` optional. It might be good to have `onChange` implemented here with maybe a `useState` in the `UserSelectFileterWithUsers` to at least allow the picker to retain the changes to selection.